### ISSUE TITLE
WebDriver executor ends each `testdriver` action with the `testharness.js` window selected

### ISSUE THREAD

**Issue by jonathan-j-lee** [web-platform-tests/wpt#45137]
See #44688 for full background.

The WebDriver implementation of `testharness.js` must switch to the `testharness.js` browsing context to run [`testharness_webdriver_resume.js`](https://github.com/web-platform-tests/wpt/blob/master/tools/wptrunner/wptrunner/executors/testharness_webdriver_resume.js). In a tabbed browser with popups in the same OS window as the `testharness.js` context, this will cause the popups to be `hidden` between `test_driver.*` calls, which may make some popup visibility interactions untestable.

CC @khushalsagar @jgraham

**Comment by jgraham** [comment]
Yes, solving this properly probably requires WebDriver-BiDi support in testdriver, so we can message navigables without actually focusing them.
